modules/combined/test/tests/catch_release/sm/catch_release_sm.i

```diff
@@ -0,0 +1,149 @@
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Mesh]
+  file = catch_release.e
+[]
+
+[Functions]
+  [./up]
+    type = PiecewiseLinear
+    x = '0 1 2.00 3 4'
+    y = '0 1 1.01 1 0'
+    scale_factor = 0.5
+  [../]
+[]
+
+[Variables]
+  [./disp_x]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+
+  [./disp_y]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+
+  [./disp_z]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[AuxVariables]
+  [./stress_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+[]
```

Comment:
again, this block and the aux kernel below would be created by the master action